### PR TITLE
{lib}[GCCcore/10.3.0] libidn2 v2.3.0

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/13ae81f07c34cd3d14680409fecc840a for a full test report.

File: easybuild/easyconfigs/l/libidn2/libidn2-2.3.0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+source_urls = ['https://ftp.gnu.org/gnu/libidn/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e1cb1db3d2e249a6a3eb6f0946777c2e892d5c5dc7bd91c74394fc3a01cab8b5']
+
+builddependencies = [('binutils', '2.36.1')]
+
+sanity_check_paths = {
+    'files': ['bin/idn2', 'lib/libidn2.%s' % SHLIB_EXT],
+    'dirs': ['include'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/l/libidn2/libidn2-2.3.0-GCCcore-10.3.0.eb:20]: @Micket Let's add a simple sanity check command here too, see https://github.com/Micket/easybuild-easyconfigs/pull/4

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3530.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/77b2c8cadd8ca76e68e22e51d0f432e7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3104.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/00b50cce29953a00d38102060353bb0d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2640.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/e35548e02db9a6ad1767716b2f2a0440 for a full test report.

**Pull request merged**